libs/providers/flagd/src/lib/configuration.ts

```diff
@@ -1,9 +1,10 @@
-import { DEFAULT_MAX_CACHE_SIZE, DEFAULT_MAX_EVENT_STREAM_RETRIES } from "./constants";
+import {DEFAULT_MAX_CACHE_SIZE, DEFAULT_MAX_EVENT_STREAM_RETRIES} from "./constants";
```

Comment:
Did your formatter remove the whitespace? This is inconsistent with other imports.